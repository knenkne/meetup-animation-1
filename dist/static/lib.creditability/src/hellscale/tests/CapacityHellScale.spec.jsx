import React from 'react'
import { shallow } from 'enzyme'

import { CapacityHellScale } from '../CapacityHellScale'

describe('<CapacityHellScale />', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<CapacityHellScale />)

        expect(wrapper).toMatchSnapshot()
    })

    it('renders correctly with used', () => {
        const wrapper = shallow(<CapacityHellScale used={50} />)

        expect(wrapper).toMatchSnapshot()
    })

    it('renders correctly with used and reserved', () => {
        const wrapper = shallow(<CapacityHellScale used={50} reserved={50} />)

        expect(wrapper).toMatchSnapshot()
    })
})
