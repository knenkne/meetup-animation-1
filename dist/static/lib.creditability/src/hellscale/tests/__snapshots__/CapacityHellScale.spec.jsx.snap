// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CapacityHellScale /> renders correctly 1`] = `
<svg
  height={136}
  viewBox="0 0 136 136"
  width={136}
  xmlns="http://www.w3.org/2000/svg"
>
  <defs>
    <Gradient
      from={0}
      id="green"
      startColor="green"
      stopColor="green"
      to={100}
    />
    <Gradient
      from={0}
      id="greenyellow"
      startColor="green"
      stopColor="yellow"
      to={43.75}
    />
    <Gradient
      from={43.75}
      id="yellowred"
      startColor="yellow"
      stopColor="red"
      to={87.5}
    />
    <Gradient
      from={87.5}
      id="red"
      startColor="red"
      stopColor="red"
      to={100}
    />
    <pattern
      height="4"
      id="diagonal"
      patternTransform="rotate(45 0 0)"
      patternUnits="userSpaceOnUse"
      width="4"
    >
      <line
        className="used"
        x1="0"
        x2="0"
        y1="0"
        y2="10"
      />
    </pattern>
  </defs>
  <g
    fill="none"
    strokeOpacity="0.3"
    strokeWidth={20}
  >
    <Arc
      from={0}
      id="greenyellow"
      to={43.75}
    />
    <Arc
      from={43.75}
      id="yellowred"
      to={87.5}
    />
    <Arc
      from={87.5}
      id="red"
      to={100}
    />
  </g>
  <g
    fill="none"
    strokeLinecap="round"
    strokeWidth={20}
  />
</svg>
`;

exports[`<CapacityHellScale /> renders correctly with used 1`] = `
<svg
  height={136}
  viewBox="0 0 136 136"
  width={136}
  xmlns="http://www.w3.org/2000/svg"
>
  <defs>
    <Gradient
      from={0}
      id="green"
      startColor="green"
      stopColor="green"
      to={100}
    />
    <Gradient
      from={0}
      id="greenyellow"
      startColor="green"
      stopColor="yellow"
      to={43.75}
    />
    <Gradient
      from={43.75}
      id="yellowred"
      startColor="yellow"
      stopColor="red"
      to={87.5}
    />
    <Gradient
      from={87.5}
      id="red"
      startColor="red"
      stopColor="red"
      to={100}
    />
    <pattern
      height="4"
      id="diagonal"
      patternTransform="rotate(45 0 0)"
      patternUnits="userSpaceOnUse"
      width="4"
    >
      <line
        className="used"
        x1="0"
        x2="0"
        y1="0"
        y2="10"
      />
    </pattern>
  </defs>
  <g
    fill="none"
    strokeOpacity="0.3"
    strokeWidth={20}
  >
    <Arc
      from={0}
      id="greenyellow"
      to={43.75}
    />
    <Arc
      from={43.75}
      id="yellowred"
      to={87.5}
    />
    <Arc
      from={87.5}
      id="red"
      to={100}
    />
  </g>
  <g
    fill="none"
    strokeLinecap="round"
    strokeWidth={20}
  >
    <Arc
      from={0}
      id="greenyellow"
      to={43.75}
    />
    <Arc
      from={43.75}
      id="yellowred"
      to={50}
    />
    <Arc
      from={0}
      id="greenyellow"
      to={1}
    />
    <Arc
      from={0}
      id="diagonal"
      to={50}
    />
  </g>
</svg>
`;

exports[`<CapacityHellScale /> renders correctly with used and reserved 1`] = `
<svg
  height={136}
  viewBox="0 0 136 136"
  width={136}
  xmlns="http://www.w3.org/2000/svg"
>
  <defs>
    <Gradient
      from={0}
      id="green"
      startColor="green"
      stopColor="green"
      to={100}
    />
    <Gradient
      from={0}
      id="greenyellow"
      startColor="green"
      stopColor="yellow"
      to={43.75}
    />
    <Gradient
      from={43.75}
      id="yellowred"
      startColor="yellow"
      stopColor="red"
      to={87.5}
    />
    <Gradient
      from={87.5}
      id="red"
      startColor="red"
      stopColor="red"
      to={100}
    />
    <pattern
      height="4"
      id="diagonal"
      patternTransform="rotate(45 0 0)"
      patternUnits="userSpaceOnUse"
      width="4"
    >
      <line
        className="used"
        x1="0"
        x2="0"
        y1="0"
        y2="10"
      />
    </pattern>
  </defs>
  <g
    fill="none"
    strokeOpacity="0.3"
    strokeWidth={20}
  >
    <Arc
      from={0}
      id="greenyellow"
      to={43.75}
    />
    <Arc
      from={43.75}
      id="yellowred"
      to={87.5}
    />
    <Arc
      from={87.5}
      id="red"
      to={100}
    />
  </g>
  <g
    fill="none"
    strokeLinecap="round"
    strokeWidth={20}
  >
    <Arc
      from={0}
      id="greenyellow"
      to={43.75}
    />
    <Arc
      from={43.75}
      id="yellowred"
      to={87.5}
    />
    <Arc
      from={87.5}
      id="red"
      to={100}
    />
    <Arc
      from={0}
      id="greenyellow"
      to={1}
    />
    <Arc
      from={0}
      id="diagonal"
      to={50}
    />
  </g>
</svg>
`;
